tests/unit/validate_request_test.py

```
@@ -0,0 +1,35 @@
+import json
+from pathlib import Path
+import sys
+
+PROJECT_ROOT = Path(__file__).resolve().parents[2]
+SRC_PATH = PROJECT_ROOT / 'src'
+sys.path.insert(0, str(SRC_PATH))
+
+from validation.validate_request import validate_request
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)